tests/README.md

```diff
@@ -1,8 +1,50 @@
 Unit tests
 ==========
 
+**WARNING**: These tests require an active Kafka cluster and will create new topics.
+
 From top-level directory run:
 
     $ tox
 
 **NOTE**: This requires `tox` ( please install with `pip install tox` ) and several supported versions of Python.
+
+Integration tests
+=================
+
+To run all of the integration test `modes` uncomment the following line from `tox.ini` and add the addresses to your Kafka and Confluent Schema Registry instances.
+
+    #python examples/integration_test.py {posargs} [yourhost:yourport] confluent-kafka-testing http://yourhost:yourport
+
+You can also run the integration tests outside of `tox` by running this command from the source root.
+
+    examples/integration_test.py <kafka-broker> [<test-topic>] [<schema-registry>]
+
+To run individual integration test `modes` with `tox` use the following syntax.
+
+    tox -- --<test mode>
+
+To get a list of modes you can run the integration test manually with the `--help` flag.
+
+    examples/integration_tests.py --help
+
+For example:
```

Comment:
Move this above the help example, otherwise one might think that this command is an example of the help.